lib/internal/modules/cjs/loader.js

```diff
@@ -413,7 +413,7 @@ function tryPackage(requestPath, exts, isMain, originalPath) {
       err.code = 'MODULE_NOT_FOUND';
       err.path = path.resolve(requestPath, 'package.json');
       err.requestPath = originalPath;
-      // TODO(BridgeAR): Add the requireStack as well.
+      err.requireStack = new Error().stack;
```

Comment:
I think @BridgeAR's comment was referring to the `requireStack` that's generated in https://github.com/nodejs/node/blob/7811d2de699d31e76b8ff9b251df68f7706b2460/lib/internal/modules/cjs/loader.js#L1027-L1032. It's different from what `new Error().stack` produces.